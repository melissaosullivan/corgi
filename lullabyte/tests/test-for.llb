/*
 * Tesing FOR statements.
 */

void main(){

    int i;
    int j;

    /*
     * Basic for loop
     */
    for (i = 0 ; i < 5 ; i = i + 1) {
        print(i);
    }
    print(42);

    /*
     * For within a for.
     */
    for (i = 0; i< 5; i=i+1) {
    	for (j=5-i; j>0; j=j-1){
    		print(i + 1);
    	}
    }
}
