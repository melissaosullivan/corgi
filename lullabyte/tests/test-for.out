0
1
2
3
4
42
1
1
1
1
1
2
2
2
2
3
3
3
4
4
5
